src/gromacs/nblib/atomtype.cpp

```diff
@@ -49,9 +49,9 @@
 namespace nblib
 {
 
-AtomType::AtomType() noexcept : name_(AtomName("")), mass_(Mass(0)), c6_(C6(0)), c12_(C12(0)) {}
+AtomType::AtomType() noexcept : name_(AtomName("")), mass_(Mass(0)), c6_(C6Param(0)), c12_(C12Param(0)) {}
```

Comment:
I think we should not use `C6Param` and `C12Param` but only `C6` and `C12`

Reply:
Yes. This was due to a conflict happening with a namesake macro defined in GROMACS.